superset/assets/stylesheets/superset.less

```diff
@@ -503,6 +503,24 @@ tr.reactable-column-header th.reactable-header-sortable {
   text-align: center;
 }
 
+.overlay-chart {
+  font-weight: bold;
+  margin-right: 10px;
```

Comment:
margin-right is not required.. Please check it.

Reply:
Done